GLConstantsTranslator/GLConstantsTranslator.html

```diff
@@ -20,18 +21,18 @@
         <p align="center">
             Other numerical inputs will be interpreted as decimal constant values. 
         </p>
-        <table align="center">
+        <table align="center" class="contenttable">
             <tr>
-                <td>Filter: </td>
-                <td>
-                <input style="font-family:'Courier'; font-size:16px" size="60" type="text" name="Input" />
+                <td class="attribute">Filter: </td>
+                <td class="content">
+                <input type="text" name="Input" class="filter" id="Filter" onclick="document.getElementById('Filter').scrollIntoView(true);" />

```

Comment:
This auto-scrolling may be ok-ish for mobile devices, but I find it somewhat annoying on the desktop. Right now, the input field seems to be at the center even on mobile devices, so I think this does not bring a noticable advantage.